cli/src/pcluster/templates/login_nodes_stack.py

```diff
@@ -0,0 +1,229 @@
+from typing import Dict
+
+from aws_cdk import aws_autoscaling as autoscaling
+from aws_cdk import aws_ec2 as ec2
+from aws_cdk import aws_elasticloadbalancingv2 as elbv2
+from aws_cdk.core import Construct, NestedStack, Stack
+
+from pcluster.config.cluster_config import SlurmClusterConfig
+from pcluster.templates.cdk_builder_utils import (
+    CdkLaunchTemplateBuilder,
+    LoginNodesIamResources,
+    get_default_instance_tags,
+    get_default_volume_tags,
+    get_login_nodes_security_groups_full,
+)
+from pcluster.utils import get_http_tokens_setting
+
+
+class Pool(Construct):
```

Comment:
The target group and load balancer are associated with a pool.
Should we create them inside the pool instead of creating them from the login nodes stack?

Reply:
Done!